packages/devtools_app/lib/src/screens/inspector/inspector_tree_controller.dart

```diff
@@ -761,6 +762,9 @@ class _InspectorTreeState extends State<InspectorTree>
 
   late DebuggerController _debuggerController;
 
+  /// When autoscrolling, the number of rows to pad the target location with.
+  final _scrollRowPadding = 3;
```

Comment:
rather than hard coding, why not use min of 3 and the number of properties? Having all properties in view was nice.